db/seeds.rb

```diff
@@ -95,20 +100,42 @@
   inventory.organization = pdx_org
 end
 
-def seed_quantity(item_name, organization_id, storage_location_id, quantity)
+DiaperDriveParticipant.create! business_name: "A Good Place to Collect Diapers",
+                               contact_name: "fred",
+                               email: "good@place.is",
+                               organization: pdx_org
+DiaperDriveParticipant.create! business_name: "A Mediocre Place to Collect Diapers",
+                               contact_name: "wilma",
+                               email: "ok@place.is",
+                               organization: pdx_org
+
+Manufacturer.find_or_create_by! name: "Manufacturer 1",
+                                organization: pdx_org
+Manufacturer.find_or_create_by! name: "Manufacturer 2",
+                                organization: pdx_org
+
+def seed_quantity(item_name, organization, storage_location, quantity)
   return if quantity == 0
 
-  item_id = Item.find_by(name: item_name, organization_id: organization_id).id
-  InventoryItem.find_or_create_by(item_id: item_id, storage_location_id: storage_location_id) do |h|
-    h.quantity = quantity
-  end
+  item = Item.find_by(name: item_name, organization: organization)
+
+  adjustment = organization.adjustments.create!(
+    comment: "Starting inventory",
+    storage_location: storage_location
+  )
+
+  LineItem.create!(quantity: quantity, item: item, itemizable: adjustment)
+
+  adjustment.reload
+  increasing_adjustment, decreasing_adjustment = adjustment.split_difference
+  adjustment.storage_location.increase_inventory increasing_adjustment
+  adjustment.storage_location.decrease_inventory decreasing_adjustment
 end
 
-# qty is Arborscape, Diaper Storage Unit, PDX Diaperbank
 items_by_category.each do |_category, entries|
   entries.each do |entry|
-    seed_quantity(entry['name'], pdx_org, inv_arbor.id, entry['qty'][0])
-    seed_quantity(entry['name'], pdx_org, inv_pdxdb.id, entry['qty'][2])
+    seed_quantity(entry['name'], pdx_org, inv_arbor, entry['qty']['arbor'])
```

Comment:
Here we gave the initial qty groups a name instead of an array offset, and updated the json fixture file.